vm/state_migration/src/type_migrations/miner/state_v10_to_v11.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::multihash::Code::Blake2b256;
+use fil_actor_miner_v10::{MinerInfo, State as MinerStateV10};
```

Comment:
Maybe make it explicit that this is version 10.

```suggestion
use fil_actor_miner_v10::{MinerInfo as MinerInfoV10, State as MinerStateV10};
```

Reply:
Fixed